packages/beacon-node/test/e2e/eth1/jsonRpcHttpClient.test.ts

```diff
@@ -153,16 +153,13 @@ describe("eth1 / jsonRpcHttpClient", function () {
       const controller = new AbortController();
       if (abort) setTimeout(() => controller.abort(), 50);
       const eth1JsonRpcClient = new JsonRpcHttpClient([url], {signal: controller.signal});
-
-      try {
-        await eth1JsonRpcClient.fetch(payload, {timeout});
-      } catch (error) {
+      await expect(eth1JsonRpcClient.fetch(payload, {timeout})).to.be.rejected.then((error) => {
         if (testCase.errorCode) {
-          expect((error as FetchError).code).to.eql(testCase.errorCode);
+          expect((error as FetchError).code).to.be.equal(testCase.errorCode);
         } else {
-          expect((error as Error).message).includes(testCase.error);
+          expect((error as Error).message).to.include(testCase.error);
         }
-      }
+      });
```

Comment:
I believe we discussed to no use the `nested then/catch` and prefer `try/catch`.

Reply:
I was not part of that discussion but in my opinion try/catch in tests should be last resort if there are no other options to use built-in assertion syntax